src/key.cpp

```diff
@@ -272,21 +272,23 @@ bool CKey::SignCompact(const uint256 &hash, std::vector<unsigned char>& vchSig)
 bool CKey::SignSchnorr(const uint256& hash, Span<unsigned char> sig, const uint256* merkle_root, const uint256& aux) const
 {
     assert(sig.size() == 64);
+    bool ret = true;
```

Comment:
nit: we could move this down, after the first early return, since it's not needed before that

Reply:
Removed all of the instances of initialisation the bool outside of a function call.